app/javascript/profileDropdown/flagButton.js

```diff
@@ -1,32 +1,18 @@
 /* global userData */
-/* eslint-disable no-alert */
+/* eslint-disable no-alert, import/order */
+import { request } from '@utilities/http';
+import { getUserDataAndCsrfToken } from '../chat/util';
+
 /**
```

Comment:
the comment block here should be relocated to the initFlag function it describes. Will fix.